Cargo.toml

```diff
@@ -248,13 +248,16 @@ bollard = { version = "0.13.0", default-features = false, features = ["ssl", "ch
 bytes = { version = "1.3.0", default-features = false, features = ["serde"] }
 bytesize = { version = "1.1.0", default-features = false }
 chrono = { version = "0.4.22", default-features = false, features = ["serde"] }
+chrono-tz = { version = "0.6.3", default-features = false, features = ["serde"], optional = true }
 cidr-utils = { version = "0.5.10", default-features = false }
 clap = { version = "4.0.32", default-features = false, features = ["derive", "error-context", "env", "help", "std", "string", "usage", "wrap_help"] }
+clickhouse-rs = { git = "https://github.com/suharev7/clickhouse-rs", rev="e40016b", features = ["tokio_io", "tls"], optional = true }
```

Comment:
Yes, I will put the comment. As far as I see, we choose a specific version since the upstream library has no tags or fresh crates.io releases. So the original author decided just to choose the specific commit.

Added a comment.